### PR TITLE
Hotfix pyodide

### PR DESCRIPTION
**Pull request by haakonvt**
### Fixed
- [Pyodide/WASM only] The iteration method for raw rows now yields rows _while running_ (instead of waiting for tasks to finish first).

### CODE REVIEW

File: cognite/client/_api/raw.py [hunk 1]
```diff
@@ -358,7 +360,7 @@ def __call__(
         Returns:
             Iterator[Row] | Iterator[RowList]: An iterator yielding the requested row or rows.
         """
-        if partitions is None:
+        if partitions is None or _RUNNING_IN_BROWSER:
```
Comment (doctrino) [inline cognite/client/_api/raw.py:363]: Is this the main bugfix? Skip concurrency if pyodide

File: cognite/client/_api/raw.py [hunk 2]
```diff
@@ -428,7 +430,7 @@ def exhaust(iterator: Iterator, results: deque[RowList], quit_early: threading.E
 
         quit_early = threading.Event()
         results: deque[RowList] = deque()  # fifo, not that ordering matters anyway...
-        pool = ConcurrencySettings.get_executor(max_workers=self._config.max_workers)
+        pool = ConcurrencySettings.get_thread_pool_executor_or_raise(max_workers=self._config.max_workers)
```
Comment (doctrino) [inline cognite/client/_api/raw.py:433]: And this is just nicer error handling?

**Review by doctrino — COMMENTED** [review]
Just a few questions for my understanding.

File: cognite/client/_api/raw.py (hunk 1 above)
Comment (haakonvt) [reply cognite/client/_api/raw.py:363]: Yup! Correct!

**Review by doctrino — APPROVED** [review]

**Pull request merged**